source/MaterialXRender/ImageHandler.h

```diff
@@ -307,6 +307,20 @@ class ImageHandler
         return -1;
     }
 
+    /// Resolve file path with any udim qualifier
+    /// @param filePath File path to resolve
+    /// @param udimString udim string replacement
+    /// @param udimTile Returned tile location
+    /// @returns Resolved file path
+    static FilePath getResolveUDIMInformation(const FilePath& filePath, const string& udimString, vector<int>& udimTile);
```

Comment:
Perhaps "getResolve**dUdim**Information"?

Reply:
Or even "resolveUdimTiles"?

Reply:
As a general comment, whenever we do need to use output arguments, it can be very confusing if they can be found anywhere in the argument list. So for example the C++ standard library always groups them at the beginning of the argument list.